sources/osg/WebGLCaps.js

```diff
@@ -390,6 +390,42 @@ WebGLCaps.prototype = {
         return this._webGLExtensions;
     },
 
+    _checkShader: function(gl, t, text) {
+        var type = t;
+        if (typeof type === 'string') type = gl[t];
+        var shader = gl.createShader(type);
+
+        gl.shaderSource(shader, text);
+        gl.compileShader(shader);
+
+        if (!gl.getShaderParameter(shader, gl.COMPILE_STATUS) && !gl.isContextLost()) {
+            console.warn("Could't compile shader " + text);
+            console.warn(gl.getShaderInfoLog(shader));
+            console.warn('You may want to consider a browser that fully supports WebGL.');
```

Comment:
Maybe the browser is not the culprit

Reply:
You're right, in fact logs wouldn't help at all in this case.
I will remove them